File: src/main/java/com/github/graqr/threshr/model/redsky/products/Author.java
package com.github.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Author(
        String nickname,
        @JsonProperty("external_id")
        String externalId) {
}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/Author.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Author(
    val nickname: String, @param:JsonProperty("external_id") val externalId: String
) {}
